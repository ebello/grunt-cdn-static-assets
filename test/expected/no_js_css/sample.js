function sample() {
  var hello = 'world';
  var asset = '//cdn.example.com/300pxPNG.png';
  var css = 'sample.css';
  var anotherAsset = 'does-not-exist.png';
  return asset;
}
