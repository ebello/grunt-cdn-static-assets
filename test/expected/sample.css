body {
  background-color: #fff;
  background-image: url('data:/rgjoierfjeori');
  color: #000;
}
div {
  background-image: url('//cdn.example.com/subfolder/Placeholder.svg');
}
p {
  text-transform: uppercase;
  background-image: url('does-not-exist.png');
}
