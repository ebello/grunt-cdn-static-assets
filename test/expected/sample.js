function sample() {
  var hello = 'world';
  var asset = '//cdn.example.com/56.png';
  var anotherAsset = 'does-not-exist.png';
  return asset;
}
