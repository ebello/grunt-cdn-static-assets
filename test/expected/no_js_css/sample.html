<html>
  <head>
    <style>
      body {
        background-image: url('//cdn.example.com/300pxJPEG.jpg');
        background-image: url('//cdn.example.com/300pxJPEG.jpg');
        background-image: url('//cdn.example.com/subfolder/Placeholder.svg');
        background-image: url('300pxJPEG/does-not-exist.jpg');
      }
    </style>
    <link href="/sample.css" rel="stylesheet" type="text/css" />
    <link href="does-not-exist.css" rel="stylesheet" type="text/css" />
    <link rel="icon" type="image/x-icon" href="//cdn.example.com/subfolder/testing.ico">
  </head>
  <body>
    <h1>Hello World</h1>
    <img src=foo/hello.png data-foo="bar"/>
    <img src="//cdn.example.com/300.px.gif"/>
    <img src="foo.png"/>
    <img src="//cdn.example.com/300pxPNG.png" data-width="900"/>
    <img src=//cdn.example.com/300pxPNG.png>
    <picture>
      <img srcset="//cdn.example.com/300pxJPEG.jpg, //cdn.example.com/subfolder/Placeholder.svg 2x" alt="Placeholder image">
    </picture>

    <script src="sample.js"></script>
    <script src="does-not-exist.js"></script>
  </body>
</html>
